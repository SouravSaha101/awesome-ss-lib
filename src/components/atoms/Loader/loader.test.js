// // @flow
// /* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
// import React from 'react'
// import { MLCAddSnapshotting, MLCTestsSetup, MLCThemes, render } from 'testUtils'

// // components.
// import Loader from '.'

// // setup.
// MLCTestsSetup()
// MLCAddSnapshotting()

// describe('Loader', () => {
//   test('is mounting and operating correctly', async () => {
//     const { getByTestId } = render(<Loader />, {
//       theme: MLCThemes[0],
//     })
//     expect(getByTestId('loader')).toMatchSnapshot(`${MLCThemes[0]}-loader`)
//   })

//   test('is displaying correctly for panel type', async () => {
//     const { getByTestId } = render(<Loader type="panel" />, {
//       theme: MLCThemes[0],
//     })
//     expect(getByTestId('loader')).toMatchSnapshot(`${MLCThemes[0]}-loader--panel`)
//   })

//   test('is displaying correctly for tab type', async () => {
//     const { getByTestId } = render(<Loader type="tab" />, {
//       theme: MLCThemes[0],
//     })
//     expect(getByTestId('loader')).toMatchSnapshot(`${MLCThemes[0]}-loader--tab`)
//   })
// })
