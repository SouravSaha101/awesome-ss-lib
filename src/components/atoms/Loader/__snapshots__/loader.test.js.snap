// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader is displaying correctly for panel type: advisor-portal-loader--panel 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  height: 100%;
  min-height: 80px;
  background-color: transparent;
}

<div
  class="emotion-0 emotion-1"
  data-testid="loader"
  type="panel"
>
  <span
    style="display: inline-block; position: relative; width: 40px; height: 40px; vertical-align: middle; font-size: 0; animation: __react-md-spinner-animation__root-rotate 1568.235294117647ms linear infinite;"
  >
    <span
      style="box-sizing: border-box; display: block; position: absolute; width: 100%; height: 100%; border-color: #00b0b9; white-space: nowrap; opacity: 1; animation-name: __react-md-spinner-animation__fill-unfill-rotate, __react-md-spinner-animation__layer-1-fade-in-out; animation-duration: 5332ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite;"
    >
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: 0px; transform: rotate(129deg); animation-name: __react-md-spinner-animation__left-spin; border-color: #00b0b9 transparent transparent #00b0b9;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: -100%; transform: rotate(-129deg); animation-name: __react-md-spinner-animation__right-spin; border-color: #00b0b9 #00b0b9 transparent transparent;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; left: 45%; width: 10%; border-width: 3px; border-top-style: solid;"
      />
    </span>
    <span
      style="box-sizing: border-box; display: block; position: absolute; width: 100%; height: 100%; border-color: #a16803; white-space: nowrap; opacity: 1; animation-name: __react-md-spinner-animation__fill-unfill-rotate, __react-md-spinner-animation__layer-2-fade-in-out; animation-duration: 5332ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite;"
    >
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: 0px; transform: rotate(129deg); animation-name: __react-md-spinner-animation__left-spin; border-color: #a16803 transparent transparent #a16803;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: -100%; transform: rotate(-129deg); animation-name: __react-md-spinner-animation__right-spin; border-color: #a16803 #a16803 transparent transparent;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; left: 45%; width: 10%; border-width: 3px; border-top-style: solid;"
      />
    </span>
    <span
      style="box-sizing: border-box; display: block; position: absolute; width: 100%; height: 100%; border-color: #ce0058; white-space: nowrap; opacity: 1; animation-name: __react-md-spinner-animation__fill-unfill-rotate, __react-md-spinner-animation__layer-3-fade-in-out; animation-duration: 5332ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite;"
    >
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: 0px; transform: rotate(129deg); animation-name: __react-md-spinner-animation__left-spin; border-color: #ce0058 transparent transparent #ce0058;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: -100%; transform: rotate(-129deg); animation-name: __react-md-spinner-animation__right-spin; border-color: #ce0058 #ce0058 transparent transparent;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; left: 45%; width: 10%; border-width: 3px; border-top-style: solid;"
      />
    </span>
    <span
      style="box-sizing: border-box; display: block; position: absolute; width: 100%; height: 100%; border-color: rgb(76, 175, 80); white-space: nowrap; opacity: 1; animation-name: __react-md-spinner-animation__fill-unfill-rotate, __react-md-spinner-animation__layer-4-fade-in-out; animation-duration: 5332ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite;"
    >
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: 0px; transform: rotate(129deg); animation-name: __react-md-spinner-animation__left-spin; border-color: rgb(76, 175, 80) transparent transparent rgb(76, 175, 80);"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: -100%; transform: rotate(-129deg); animation-name: __react-md-spinner-animation__right-spin; border-color: rgb(76, 175, 80) rgb(76, 175, 80) transparent transparent;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; left: 45%; width: 10%; border-width: 3px; border-top-style: solid;"
      />
    </span>
  </span>
</div>
`;

exports[`Loader is displaying correctly for tab type: advisor-portal-loader--tab 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  min-height: 280px;
  background-color: #FFFFFF;
}

<div
  class="emotion-0 emotion-1"
  data-testid="loader"
  type="tab"
>
  <span
    style="display: inline-block; position: relative; width: 40px; height: 40px; vertical-align: middle; font-size: 0; animation: __react-md-spinner-animation__root-rotate 1568.235294117647ms linear infinite;"
  >
    <span
      style="box-sizing: border-box; display: block; position: absolute; width: 100%; height: 100%; border-color: #00b0b9; white-space: nowrap; opacity: 1; animation-name: __react-md-spinner-animation__fill-unfill-rotate, __react-md-spinner-animation__layer-1-fade-in-out; animation-duration: 5332ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite;"
    >
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: 0px; transform: rotate(129deg); animation-name: __react-md-spinner-animation__left-spin; border-color: #00b0b9 transparent transparent #00b0b9;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: -100%; transform: rotate(-129deg); animation-name: __react-md-spinner-animation__right-spin; border-color: #00b0b9 #00b0b9 transparent transparent;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; left: 45%; width: 10%; border-width: 3px; border-top-style: solid;"
      />
    </span>
    <span
      style="box-sizing: border-box; display: block; position: absolute; width: 100%; height: 100%; border-color: #a16803; white-space: nowrap; opacity: 1; animation-name: __react-md-spinner-animation__fill-unfill-rotate, __react-md-spinner-animation__layer-2-fade-in-out; animation-duration: 5332ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite;"
    >
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: 0px; transform: rotate(129deg); animation-name: __react-md-spinner-animation__left-spin; border-color: #a16803 transparent transparent #a16803;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: -100%; transform: rotate(-129deg); animation-name: __react-md-spinner-animation__right-spin; border-color: #a16803 #a16803 transparent transparent;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; left: 45%; width: 10%; border-width: 3px; border-top-style: solid;"
      />
    </span>
    <span
      style="box-sizing: border-box; display: block; position: absolute; width: 100%; height: 100%; border-color: #ce0058; white-space: nowrap; opacity: 1; animation-name: __react-md-spinner-animation__fill-unfill-rotate, __react-md-spinner-animation__layer-3-fade-in-out; animation-duration: 5332ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite;"
    >
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: 0px; transform: rotate(129deg); animation-name: __react-md-spinner-animation__left-spin; border-color: #ce0058 transparent transparent #ce0058;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: -100%; transform: rotate(-129deg); animation-name: __react-md-spinner-animation__right-spin; border-color: #ce0058 #ce0058 transparent transparent;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; left: 45%; width: 10%; border-width: 3px; border-top-style: solid;"
      />
    </span>
    <span
      style="box-sizing: border-box; display: block; position: absolute; width: 100%; height: 100%; border-color: rgb(76, 175, 80); white-space: nowrap; opacity: 1; animation-name: __react-md-spinner-animation__fill-unfill-rotate, __react-md-spinner-animation__layer-4-fade-in-out; animation-duration: 5332ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite;"
    >
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: 0px; transform: rotate(129deg); animation-name: __react-md-spinner-animation__left-spin; border-color: rgb(76, 175, 80) transparent transparent rgb(76, 175, 80);"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: -100%; transform: rotate(-129deg); animation-name: __react-md-spinner-animation__right-spin; border-color: rgb(76, 175, 80) rgb(76, 175, 80) transparent transparent;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; left: 45%; width: 10%; border-width: 3px; border-top-style: solid;"
      />
    </span>
  </span>
</div>
`;

exports[`Loader is mounting and operating correctly: advisor-portal-loader 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  min-height: 80px;
  background-color: transparent;
}

<div
  class="emotion-0 emotion-1"
  data-testid="loader"
  type="component"
>
  <span
    style="display: inline-block; position: relative; width: 40px; height: 40px; vertical-align: middle; font-size: 0; animation: __react-md-spinner-animation__root-rotate 1568.235294117647ms linear infinite;"
  >
    <span
      style="box-sizing: border-box; display: block; position: absolute; width: 100%; height: 100%; border-color: #00b0b9; white-space: nowrap; opacity: 1; animation-name: __react-md-spinner-animation__fill-unfill-rotate, __react-md-spinner-animation__layer-1-fade-in-out; animation-duration: 5332ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite;"
    >
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: 0px; transform: rotate(129deg); animation-name: __react-md-spinner-animation__left-spin; border-color: #00b0b9 transparent transparent #00b0b9;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: -100%; transform: rotate(-129deg); animation-name: __react-md-spinner-animation__right-spin; border-color: #00b0b9 #00b0b9 transparent transparent;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; left: 45%; width: 10%; border-width: 3px; border-top-style: solid;"
      />
    </span>
    <span
      style="box-sizing: border-box; display: block; position: absolute; width: 100%; height: 100%; border-color: #a16803; white-space: nowrap; opacity: 1; animation-name: __react-md-spinner-animation__fill-unfill-rotate, __react-md-spinner-animation__layer-2-fade-in-out; animation-duration: 5332ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite;"
    >
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: 0px; transform: rotate(129deg); animation-name: __react-md-spinner-animation__left-spin; border-color: #a16803 transparent transparent #a16803;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: -100%; transform: rotate(-129deg); animation-name: __react-md-spinner-animation__right-spin; border-color: #a16803 #a16803 transparent transparent;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; left: 45%; width: 10%; border-width: 3px; border-top-style: solid;"
      />
    </span>
    <span
      style="box-sizing: border-box; display: block; position: absolute; width: 100%; height: 100%; border-color: #ce0058; white-space: nowrap; opacity: 1; animation-name: __react-md-spinner-animation__fill-unfill-rotate, __react-md-spinner-animation__layer-3-fade-in-out; animation-duration: 5332ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite;"
    >
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: 0px; transform: rotate(129deg); animation-name: __react-md-spinner-animation__left-spin; border-color: #ce0058 transparent transparent #ce0058;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: -100%; transform: rotate(-129deg); animation-name: __react-md-spinner-animation__right-spin; border-color: #ce0058 #ce0058 transparent transparent;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; left: 45%; width: 10%; border-width: 3px; border-top-style: solid;"
      />
    </span>
    <span
      style="box-sizing: border-box; display: block; position: absolute; width: 100%; height: 100%; border-color: rgb(76, 175, 80); white-space: nowrap; opacity: 1; animation-name: __react-md-spinner-animation__fill-unfill-rotate, __react-md-spinner-animation__layer-4-fade-in-out; animation-duration: 5332ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite;"
    >
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: 0px; transform: rotate(129deg); animation-name: __react-md-spinner-animation__left-spin; border-color: rgb(76, 175, 80) transparent transparent rgb(76, 175, 80);"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: relative; width: 50%; height: 100%; overflow: hidden;"
      >
        <span
          style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; bottom: 0px; width: 200%; border-width: 3px; border-style: solid; animation-duration: 1333ms; animation-timing-function: cubic-bezier(.4, 0, .2, 1); animation-iteration-count: infinite; left: -100%; transform: rotate(-129deg); animation-name: __react-md-spinner-animation__right-spin; border-color: rgb(76, 175, 80) rgb(76, 175, 80) transparent transparent;"
        />
      </span>
      <span
        style="display: inline-block; box-sizing: border-box; position: absolute; top: 0px; border-radius: 50%; left: 45%; width: 10%; border-width: 3px; border-top-style: solid;"
      />
    </span>
  </span>
</div>
`;
